src/dodal/devices/slits/gap_and_centre_slit_base_classes.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC, abstractmethod
+from typing import Tuple
+
+from bluesky.protocols import Movable
+from ophyd import Device
+
+
+class GapAndCentreSlit1d(Device, Movable, ABC):
+    """Class defining interface between Ophyd and 1-dimensional gap and centre slits."""
+
+    @abstractmethod
+    def set(self, position_tuple: Tuple[float, float]):
+        """Method to set position of slit.
```

Comment:
nit: is it worth mentioning that the x values here are in a virtual coordinate system?